src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -140,7 +143,13 @@ public function setupFieldAssociations($fields, $table, $model, $strategy)
 
         foreach ($fields as $field) {
             $name = $alias . '_' . $field . '_translation';
-
+            $conditions = [
+                $name . '.model' => $fieldConditions['model'],
+                $name . '.field' => $field,
+            ];
+            foreach ($fieldConditions as $fieldName => $fieldValue) {
+                $conditions[$name . '.' . $fieldName] = $fieldValue;
```

Comment:
Why are we manipulating the conditions here? I would expect the end user to provide conditions with aliases. This would be one of the few places we don't expect people to put aliases on conditions.